src/FffGcodeWriter.cpp

```diff
@@ -1177,9 +1177,9 @@ void FffGcodeWriter::addMeshLayerToGCode(const SliceDataStorage& storage, const
     }
 
     const ExtruderTrain* train = storage.meshgroup->getExtruderTrain(extruder_nr);
-
-    gcode_layer.setMesh(reinterpret_cast<size_t>(&mesh));
-
+    //PJP
```

Comment:
What does PJP stand for? Why is this here?

Reply:
This was just a place holder for me to do something when I made some changes and was not in an IDE. I will remove.